pyomo/contrib/gdpopt/tests/test_gdpopt.py

```diff
@@ -590,34 +590,34 @@ def test_infeasible_GDP(self):
             self.assertIn("Set covering problem was infeasible.",
                           output.getvalue().strip())
 
-    def test_GDP_nonlinear_objective(self):
-        m = ConcreteModel()
-        m.x = Var(bounds=(-1, 10))
-        m.y = Var(bounds=(2, 3))
-        m.d = Disjunction(expr=[
-            [m.x + m.y >= 5], [m.x - m.y <= 3]
-        ])
-        m.o = Objective(expr=m.x ** 2)
-        SolverFactory('gdpopt').solve(
-            m, strategy='RIC',
-            mip_solver=mip_solver,
-            nlp_solver=nlp_solver
-        )
-        self.assertAlmostEqual(value(m.o), 0)
-
-        m = ConcreteModel()
-        m.x = Var(bounds=(-1, 10))
-        m.y = Var(bounds=(2, 3))
-        m.d = Disjunction(expr=[
-            [m.x + m.y >= 5], [m.x - m.y <= 3]
-        ])
-        m.o = Objective(expr=-m.x ** 2, sense=maximize)
-        SolverFactory('gdpopt').solve(
-            m, strategy='RIC',
-            mip_solver=mip_solver,
-            nlp_solver=nlp_solver
-        )
-        self.assertAlmostEqual(value(m.o), 0)
+    # def test_GDP_nonlinear_objective(self):
```

Comment:
Please add a comment here referring to the PR and that it will get fixed in a future PR to not loose track of this

Reply:
It is actually already fixed in #2406